script_mteb_french/results_analysis/results_parser.py

```diff
@@ -7,15 +7,20 @@
 import pandas as pd
 
 DATASET_KEYS = {
-    "DiaBLaBitextMining": ["fr-en"],
+    "DiaBlaBitextMining": ["fr-en"],
     "FloresBitextMining": MTEB(tasks=['FloresBitextMining'], task_langs=['fr', 'en']).tasks[0].langs,
     "MasakhaNEWSClassification": MTEB(tasks=['MasakhaNEWSClassification'], task_langs=['fr']).tasks[0].langs,
     "MasakhaNEWSClusteringS2S": MTEB(tasks=['MasakhaNEWSClusteringS2S'], task_langs=['fr']).tasks[0].langs,
     "MasakhaNEWSClusteringP2P": MTEB(tasks=['MasakhaNEWSClusteringP2P'], task_langs=['fr']).tasks[0].langs,
+    "XPQARetrieval": MTEB(tasks=['XPQARetrieval'], task_langs=['fr']).tasks[0].langs,
 }
 
+HF_SUBSETS_VALUES = ["fra-fra"]
+ISO3_LANGUAGE = ["fra-Latn"]
 
-MODELS_TO_IGNORE = ['voyage-01', 'voyage-02', 'voyage-lite-01']
+MODELS_TO_IGNORE = ['voyage-01', 'voyage-02', 'voyage-lite-01', 'Geotrend/distilbert-base-en-fr-es-pt-it-cased', 
+                    'Geotrend/bert-base-10lang-cased', 'Geotrend/bert-base-15lang-cased', 'Geotrend/bert-base-25lang-cased',
+                    'dangvantuan/sentence-camembert-large', 'distilbert-base-uncased']
```

Comment:
why ignoring sentence-camembert-large. It was one of the best model?

Reply:
because there is a newer version: Lajavaness/sentence-camembert-large, see description in this [link](https://huggingface.co/Lajavaness/sentence-camembert-large). they say it's better than the other one.

I see that Lajavaness proposes a [sentence-flaubert](https://huggingface.co/Lajavaness/sentence-flaubert-base), maybe we should add it instead of flaubert

Reply:
just saw that! Nice